frontend/scripts/start.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
```

Comment:
Since we already have a `start.sh`, I reckon we should rename these scripts to something like `ui_start.js`, `ui_build.js` and `ui_test.js` and then tweak said names in package.json; what do you reckon?